exercises/kindergarten-garden/src/test/kotlin/KindergartenGardenTest.kt

```diff
@@ -0,0 +1,88 @@
+import kotlin.test.Test
+import kotlin.test.Ignore
+import kotlin.test.assertEquals
+
+class KindergartenGardenTest {
+
+    @Test
+    fun singleStudent() {
```

Comment:
Let's use backticked names. There are two sets of tests at `canonical-data`: *partial garden* and *full garden*. 
Following format is suggested: `case | test`, like `parial | single student` name for the first test and `full | last student's garden` for the last one.